turbinia/config/__init__.py

```diff
@@ -47,6 +47,7 @@
     # GCE CONFIG
     'PROJECT',
     'ZONE',
+    'GCF_REGION',
```

Comment:
Can we maybe make this a TURBINIA_REGION var to make it a little more generic?   Other components are also regionalized, and we are probably just implicitly using the local region in other places too, so it would be nice to point to this same var.   Also, I wouldn't think that we would need a cross-region setup here (except for the source GCP disks, which we'll want to access remotely, but the Turbinia components will still be in the same region) any time soon, so I don't think we'll need cloud service specific vars for the region at this time.

Reply:
Done